Automattic/wp-calypso
# New Onboarding: show design previews in landscape

#### Changes proposed in this Pull Request

- Adds & enables a landscape design preview option in new onboarding
- Enables mshots design previews in all environments
- includes tweak for backing off mshots refreshes the longer it takes
- includes a scroll effect on mousehover for landscape previews

#### Testing instructions

* Check signup displays mshot images with landscape preview http://calypso.localhost:3000/new/design
* Check localized signup displays translated design preview content http://calypso.localhost:3000/new/design/ja
* Check anchor.fm signup flow displays mshot images http://calypso.localhost:3000/new/?anchor_podcast=22b6608

Landscape views:
![Screenshot_2021-01-22 WordPress com(4)](https://user-images.githubusercontent.com/811776/105461748-bf312c80-5ce1-11eb-801b-c0498f1c8158.jpg)

Hover effect:
![landscape-hover](https://user-images.githubusercontent.com/811776/105461782-c821fe00-5ce1-11eb-8679-3e8c5e1d295c.gif)



Relates #49097
